package.json

```diff
@@ -53,7 +53,7 @@
     "jsdom-testing-mocks": "^1.15.2",
     "msw": "2.10.1",
     "msw-storybook-addon": "^2.0.5",
-    "node-gyp": "^11.3.0",
+    "node-gyp": "^11.4.2",
```

Comment:
We can't get rid of node-gyp yet because pnpm still ships with node-gyp 11.1.0 which is broken on Windows (https://github.com/gravitational/teleport/pull/53330). At this point I should probably open an issue in the pnpm repo asking if they could update it.